clients/admin-ui/src/features/datastore-connections/system_portal_config/forms/ConnectorParameters.tsx

```diff
@@ -358,6 +387,24 @@ export const ConnectorParameters: React.FC<ConnectorParametersProps> = ({
         deleteResult={deleteDatastoreConnectionResult}
       />
 
+      {connectionConfig && (
```

Comment:
we've gotten into the habit of doing `test ? something : null` instead of `&&` because of the reason explained in [this article](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx). to be clear, the bug described there wouldn't come up in this case since `connectionConfig` is an object, but sometimes it's easier to just always stick with the ternary as opposed to the `&&`

Reply:
Noted!  I changed this.